test/test-remote/looker/src/index.ts

```diff
@@ -134,9 +138,8 @@ async function main() {
     // to more reliably crash looker, see CH1288. I'd love for all unexpected
     // errors to crash the runtime instead.
     try {
-      // we know better than:
-      //  error TS2454: Variable 'dummystreams' is used before being assigned.
-      // the first iteration sets dummystreams.
+      // we pass the updated cycle id here even if dummystreams doesnt use it
+      // this ensures the cycle report json has a distinct filename
       //@ts-expect-error: see comment above
       await performWriteReadCycle(cyclenum, dummystreams, guniqueCycleId);
```

Comment:
just noticing while I read this: the `gunique..` way of naming things was past JP expressing that this should be _globally_ unique, i.e. always have enough randomness in it to never collide with anything. The obvious question: is this still true?

Reply:
trying out `invocationCycleId` in followup PR